Sources/VLCSettingsController.m

```diff
@@ -125,12 +138,38 @@ - (void)updateUIAndCoreSpotlightForPasscodeSetting:(BOOL)passcodeOn
 
 #pragma mark - IASKSettings delegate
 
-- (void)settingsViewController:(IASKAppSettingsViewController*)sender buttonTappedForSpecifier:(IASKSpecifier*)specifier
+- (IBAction)pushAboutViewController:(id)sender
 {
     VLCAboutViewController *aboutVC = [[VLCAboutViewController alloc] init];
     [self.navigationController pushViewController:aboutVC animated:YES];
 }
 
+- (UIView *)settingsViewController:(id<IASKViewController>)settingsViewController tableView:(UITableView *)tableView viewForHeaderForSection:(NSInteger)section
+{
+    UIView *header = [[UIView alloc] initWithFrame:CGRectMake(20, 0, tableView.bounds.size.width, 64)];
+    UILabel *headerLabel = [[UILabel alloc] init];
+    
+    headerLabel.font = [UIFont systemFontOfSize:23 weight:UIFontWeightSemibold];
+    headerLabel.text = [settingsViewController.settingsReader titleForSection:section];
+    headerLabel.textColor = PresentationTheme.current.colors.cellTextColor;
+    headerLabel.translatesAutoresizingMaskIntoConstraints = false;
+    [header addSubview:headerLabel];
+    
+    UILayoutGuide *margins = [header layoutMarginsGuide];
+    [NSLayoutConstraint activateConstraints:@[
+                                              [headerLabel.leadingAnchor constraintEqualToAnchor:margins.leadingAnchor constant:8],
+                                              [headerLabel.trailingAnchor constraintEqualToAnchor:margins.trailingAnchor],
+                                              [headerLabel.topAnchor constraintEqualToAnchor:margins.topAnchor],
+                                              [headerLabel.bottomAnchor constraintEqualToAnchor:margins.bottomAnchor]
+                                              ]];
+    return header;
+}
+
+- (CGFloat)tableView:(UITableView *)tableView heightForRowAtIndexPath:(NSIndexPath *)indexPath
+{
+    return 64.0;
```

Comment:
What would you think about having a dynamic height?_(mostly for accessibility reasons)_
Like for example what @carolanitz  did in bf01cc4477fe420f36371605acb48d6c4a5a06fe

Reply:
That's a good idea but I'm not really sure how to do make a cell have a minimum height of 64 px - as shown in the mockups - when I'm using the default `UITableViewCellStyleSubtitle` for example. Is making my own custom cell the only option?